pandas/core/strings.py

```diff
@@ -1842,23 +1842,38 @@ def extractall(self, pat, flags=0):
         return str_extractall(self._orig, pat, flags=flags)
 
     _shared_docs['find'] = ("""
+    Retrieve least index of substring.
+
     Return %(side)s indexes in each strings in the Series/Index
     where the substring is fully contained between [start:end].
     Return -1 on failure. Equivalent to standard :meth:`str.%(method)s`.
 
     Parameters
     ----------
     sub : str
-        Substring being searched
+        Substring being searched.
     start : int
-        Left edge index
+        Left edge index.
     end : int
-        Right edge index
+        Right edge index.
 
     Returns
     -------
     found : Series/Index of integer values
 
+    Examples
+    --------
+    >>> s = pd.Series('Finding with pandas')
```

Comment:
s = pd.Series(['Finding with pandas']) just to be clearer

Reply:
Actually, could you make a series with multiple elements so that it's clear this operates on each element of the series?

Reply:
Done!

Reply:
is this update with a list of strings (and maybe some np.nan) pushed?

Reply:
@jreback  No it isnt, i will update it and push.